src/components/Sidebar/Sidebar.js

```diff
@@ -125,45 +136,80 @@ class Sidebar extends PureComponent<Props, State> {
         {({ sidebarOffsetPercentage, firstProjectPosition }) => (
           <Fragment>
             <Wrapper offset={`${sidebarOffsetPercentage}%`}>
-              <ScrollbarOnlyVertical
-                autoHide
-                renderTrackHorizontal={props => (
-                  <div {...props} style={{ display: 'none' }} />
-                )}
-              >
-                <IntroductionBlurb
-                  isVisible={!finishedOnboarding && introSequenceStepIndex >= 1}
-                />
-
-                <Projects offset={`${firstProjectPosition}px`}>
-                  {projects.map(project => (
-                    <Fragment key={project.id}>
-                      <Tooltip title={project.name} position="right">
-                        <SidebarProjectIcon
-                          size={SIDEBAR_ICON_SIZE}
-                          id={project.id}
-                          name={project.name}
-                          color={project.color}
-                          iconSrc={project.icon}
-                          isSelected={
-                            finishedOnboarding &&
-                            project.id === selectedProjectId
+              <DragDropContext onDragEnd={this.onDragEnd}>
+                <ScrollbarOnlyVertical
+                  autoHide
+                  renderTrackHorizontal={props => (
```

Comment:
`renderTrackHorizontal` can be removed by conditionally rendering `IntroductionBlurb`.